<!DOCTYPE html>
<html lang="en-US">

<head>
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">

    <link rel="stylesheet" href="/biodica/assets/css/main.css">
    <link rel="stylesheet" href="/biodica/assets/css/font-awesome.min.css">

    <link rel="shortcut icon" href="/biodica/favicon.ico?1">
    <!-- Begin Jekyll SEO tag v2.7.1 -->
<title>References | BIODICA</title>
<meta name="generator" content="Jekyll v3.9.0" />
<meta property="og:title" content="References" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Seminal ICA articles" />
<meta property="og:description" content="Seminal ICA articles" />
<link rel="canonical" href="http://localhost:4000/biodica/docs/references/" />
<meta property="og:url" content="http://localhost:4000/biodica/docs/references/" />
<meta property="og:site_name" content="BIODICA" />
<meta property="og:type" content="article" />
<meta property="article:published_time" content="2022-01-13T00:05:34+01:00" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="References" />
<script type="application/ld+json">
{"@type":"WebPage","headline":"References","dateModified":"2022-01-13T00:05:34+01:00","datePublished":"2022-01-13T00:05:34+01:00","url":"http://localhost:4000/biodica/docs/references/","description":"Seminal ICA articles","@context":"https://schema.org"}</script>
<!-- End Jekyll SEO tag -->


    <link rel="alternate" type="application/rss+xml" title="BIODICA" href="http://localhost:4000/biodica/feed.xml" />
</head>


<body>

    <nav class="navbar navbar-default navbar-fixed-top">
    <div class="container navbar-container">
        <div class="navbar-header">
            <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
            <a class="navbar-brand" href="/biodica/">
                <i class="fa fa-home"></i> BIODICA
            </a>
        </div>
        <div id="navbar" class="collapse navbar-collapse">
            <ul class="nav navbar-nav">
                <li  class="active" ><a href="/biodica/docs/home/">Docs</a></li>
                <li ><a href="/biodica/install/">Install</a></li>
                <li ><a href="/biodica/news/2022/01/04/welcome-to-biodica/">News</a></li>
                <li ><a href="/biodica/about_us/">About us</a></li>
            </ul>
            <div class="navbar-right">
                <form class="navbar-form navbar-left">
                    <div class="form-group has-feedback">
                        <input id="search-box" type="search" class="form-control" placeholder="Search...">
                        <i class="fa fa-search form-control-feedback"></i>
                    </div>
                </form>
                <ul class="nav navbar-nav">
                    <li><a href="https://github.com/LabBandSB/BIODICA"><i class="fa fa-github" aria-hidden="true"></i></a></li>
                </ul>
            </div>
        </div>
    </div>
</nav>


    <div class="page-content">
        <div class="wrapper">
            <div class="container">
    <div class="row">
        <div class="col-md-4">
          <div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
  
    <div class="panel panel-default">
      <div class="panel-heading">
        <h4 class="panel-title">
          <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapse-1" aria-expanded="false" aria-controls="collapse-1">
            Getting Started
          </a>
        </h4>
      </div>
      
      
      <div class="panel-collapse" id="collapse-1" role="tabpanel" aria-label="Side Navigation">
        <div class="list-group">
          
            
            
            <a class="list-group-item " href="/biodica/docs/home/">Welcome</a>
          
            
            
            <a class="list-group-item " href="/biodica/docs/tutorials/">Tutorials</a>
          
            
            
            <a class="list-group-item active" href="/biodica/docs/references/">References</a>
          
        </div>
      </div>
    </div>
  
</div>

        </div>

        <div class="col-md-8">
            <h2>References</h2>
            <div id="markdown-content-container" style="font-size:16px; width: 100%;
            height:100%"><h3 id="seminal-ica-articles">Seminal ICA articles</h3>

<ul>
  <li>
    <p>Bell, A.J.; Sejnowski, T.J. An information-maximization approach to blind separation and blind deconvolution. Neural Comput. 1995, 7, 1129–1159. <a href="https://direct.mit.edu/neco/article-abstract/7/6/1129/5909/An-Information-Maximization-Approach-to-Blind?redirectedFrom=fulltext">Link</a></p>
  </li>
  <li>
    <p>Hyvärinen, A.; Oja, E. Independent component analysis: Algorithms and applications. Neural Netw. 2000, 13, 411–430. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0893608000000265">Link</a></p>
  </li>
  <li>
    <p>Himberg, J.; Hyvarinen, A. Icasso: Software for investigating the reliability of ICA estimates by clustering
and visualization. In Proceedings of the 2003 IEEE XIII Workshop on Neural Networks for Signal Processing (IEEE Cat. No.03TH8718), Toulouse, France, 17–19 September 2003; pp. 259–268. <a href="https://ieeexplore.ieee.org/document/1318025">Link</a></p>
  </li>
</ul>

<h3 id="first-studies-applying-ica-to-gene-expression-data-analysis">First studies applying ICA to gene expression data analysis</h3>

<ul>
  <li>
    <p>Liebermeister, W. Linear modes of gene expression determined by independent component analysis. Bioinformatics 2002, 18, 51-60. <a href="https://academic.oup.com/bioinformatics/article/18/1/51/243636">Link</a></p>
  </li>
  <li>
    <p>Lee, S.-I.; Batzoglou, S. Application of independent component analysis to microarrays. Genome Biol. 2003, 4, R76. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2003-4-11-r76">Link</a></p>
  </li>
  <li>
    <p>Saidi, S.A.; Holland, C.M.; Kreil, D.P.;MacKay, D.J.C.; Charnock-Jones, D.S.; Print, C.G.; Smith, S.K. Independent component analysis of microarray data in the study of endometrial cancer. Oncogene 2004, 23, 6677-6683. <a href="https://www.nature.com/articles/1207562">Link</a></p>
  </li>
  <li>
    <p>Teschendorff, A.E.; Journee, M.; Absil, P.A.; Sepulchre, R.; Caldas, C. Elucidating the altered transcriptional programs in breast cancer using independent component analysis. PLoS Comput. Biol. 2007, 3, e161. <a href="https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030161">Link</a></p>
  </li>
</ul>

<h3 id="review-articles">Review articles</h3>

<ul>
  <li>
    <p>Kong, W.; Vanderburg, C.R.; Gunshin, H.; Rogers, J.T.; Huang, X. A review of independent component analysis application to microarray gene expression data. Biotechniques 2008, 45, 501–520. <a href="https://www.future-science.com/doi/10.2144/000112950">Link</a></p>
  </li>
  <li>
    <p>Zinovyev, A.; Kairov, U.; Karpenyuk, T.; Ramanculov, E. Blind source separation methods for deconvolution of complex signals in cancer biology. Biochem. Biophys. Res. Commun. 2013, 430, 1182–1187. <a href="https://www.sciencedirect.com/science/article/abs/pii/S0006291X12023741?via%3Dihub">Link</a></p>
  </li>
  <li>
    <p>Sompairac N, Nazarov PV, Czerwinska U, Cantini L, Biton A, Molkenov A, Zhumadilov Z, Barillot E, Radvanyi F, Gorban A, Kairov U, Zinovyev A. Independent Component Analysis for Unraveling the Complexity of Cancer Omics Datasets. Int J Mol Sci. 2019 Sep 7;20(18):4414. doi: 10.3390/ijms20184414. <a href="https://www.mdpi.com/1422-0067/20/18/4414">Link</a></p>
  </li>
</ul>

<h3 id="comparison-between-ica-and-other-matrix-factorization-methods">Comparison between ICA and other matrix factorization methods</h3>

<ul>
  <li>
    <p>Cantini, L.; Kairov, U.; de Reyniès, A.; Barillot, E.; Radvanyi, F.; Zinovyev, A. Assessing reproducibility of matrix factorization methods in independent transcriptomes. Bioinformatics 2019. <a href="https://academic.oup.com/bioinformatics/article/35/21/4307/5426054">Link</a></p>
  </li>
  <li>
    <p>Way GP, Zietz M, Rubinetti V, Himmelstein DS, Greene CS. Compressing gene expression data using multiple latent space dimensionalities learns complementary biological representations. Genome Biol. 2020 May 11;21(1):109. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-020-02021-3">Link</a></p>
  </li>
</ul>

<h3 id="meta-analysis-of-multiple-datasets-using-ica">Meta-analysis of multiple datasets using ICA</h3>

<ul>
  <li>Biton, A.; Bernard-Pierrot, I.; Lou, Y.; Krucker, C.; Chapeaublanc, E.; Rubio-Pérez, C.; López-Bigas, N.; Kamoun, A.; Neuzillet, Y.; Gestraud, P.; et al. Independent component analysis uncovers the landscape of the bladder tumor transcriptome and reveals insights into luminal and basal subtypes. Cell Rep. 2014, 9, 1235–1245. <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(14)00904-8?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124714009048%3Fshowall%3Dtrue">Link</a></li>
</ul>

<h3 id="choosing-the-right-number-of-ica-components-in-omics-studies">Choosing the right number of ICA components in omics studies</h3>

<ul>
  <li>
    <p>Kairov, U.; Cantini, L.; Greco, A.; Molkenov, A.; Czerwinska, U.; Barillot, E.; Zinovyev, A. Determining the optimal number of independent components for reproducible transcriptomic data analysis. BMC Genom. 2017, 18, 712. <a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/s12864-017-4112-9">Link</a></p>
  </li>
  <li>
    <p>Krumsiek, J.; Suhre, K.; Illig, T.; Adamski, J.; Theis, F.J. Bayesian Independent Component Analysis Recovers Pathway Signatures from Blood Metabolomics Data. J. Proteome Res. 2012, 11, 4120–4131. <a href="https://pubs.acs.org/doi/10.1021/pr300231n">Link</a></p>
  </li>
</ul>

<h3 id="applications-to-single-cell-data-analysis">Applications to single cell data analysis</h3>

<ul>
  <li>
    <p>Aynaud MM, Mirabeau O, Gruel N, Grossetête S, Boeva V, Durand S, Surdez D, Saulnier O, Zaïdi S, Gribkova S, Fouché A, Kairov U, Raynal V, Tirode F, Grünewald TGP, Bohec M, Baulande S, Janoueix-Lerosey I, Vert JP, Barillot E, Delattre O, Zinovyev A. Transcriptional Programs Define Intratumoral Heterogeneity of Ewing Sarcoma at Single-Cell Resolution. Cell Rep. 2020 Feb 11;30(6):1767-1779.e6. <a href="https://www.cell.com/cell-reports/fulltext/S2211-1247(20)30074-7?_returnURL=https%3A%2F%2Flinkinghub.elsevier.com%2Fretrieve%2Fpii%2FS2211124720300747%3Fshowall%3Dtrue">Link</a></p>
  </li>
  <li>
    <p>Kondratova M, Czerwinska U, Sompairac N, Amigorena SD, Soumelis V, Barillot E, Zinovyev A, Kuperstein I. A multiscale signalling network map of innate immune response in cancer reveals cell heterogeneity signatures. Nat Commun. 2019 Oct 22;10(1):4808. <a href="https://www.nature.com/articles/s41467-019-12270-x">Link</a></p>
  </li>
</ul>

<h3 id="applications-of-ica-to-other-than-gene-expression-omics-data-and-multiomics">Applications of ICA to other than gene expression omics data and multiomics</h3>

<ul>
  <li>
    <p>Scherer M, Nazarov PV, Toth R, Sahay S, Kaoma T, Maurer V, Vedeneev N, Plass C, Lengauer T, Walter J, Lutsik P. Reference-free deconvolution, visualization and interpretation of complex DNA methylation data using DecompPipeline, MeDeCom and FactorViz. Nat Protoc. 2020 Oct;15(10):3240-3263. <a href="https://www.nature.com/articles/s41596-020-0369-6">Link</a></p>
  </li>
  <li>
    <p>Meunier L, Hirsch TZ, Caruso S, Imbeaud S, Bayard Q, Roehrig A, Couchy G, Nault JC, Llovet JM, Blanc JF, Calderaro J, Zucman-Rossi J, Letouzé E. DNA Methylation Signatures Reveal the Diversity of Processes Remodeling Hepatocellular Carcinoma Methylomes. Hepatology. 2021 Aug;74(2):816-834. <a href="https://aasldpubs.onlinelibrary.wiley.com/doi/10.1002/hep.31796">Link</a></p>
  </li>
  <li>
    <p>Teschendorff, A.E.; Jing, H.; Paul, D.S.; Virta, J.; Nordhausen, K. Tensorial blind source separation for improved analysis of multi-omic data. Genome Biol. 2018, 19, 76. <a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-018-1455-8">Link</a></p>
  </li>
</ul>
</div>
            <hr>
            





  
  

  
  

  
  
    <ul class="pager">
      
        
        
        <li class="previous">
          <a href="/biodica/docs/tutorials/">
            <span aria-hidden="true">&larr;</span> Previous
          </a>
        </li>
      

      
        <li class="next disabled">
          <a>
            Next <span aria-hidden="true">&rarr;</span>
          </a>
        </li>
      
      </ul>
    <div class="clear"></div>
    

        </div>

    </div>
</div>

        </div>
    </div>

    <footer class="footer">
    <div class="container">

        <p class="text-center">
            BIODICA 2022 |
            Powered by <a href="https://github.com/aksakalli/jekyll-doc-theme">Jekyll Doc Theme</a>
        </p>
        <!-- <p class="text-muted">Place sticky footer content here.</p> -->
    </div>
</footer>

    <script>
  var baseurl = '/biodica'
</script>
<script src="https://code.jquery.com/jquery-1.12.4.min.js"></script>
<script src="/biodica/assets/js/bootstrap.min.js "></script>
<script src="/biodica/assets/js/typeahead.bundle.min.js "></script>

<script src="/biodica/assets/js/main.js "></script>

</body>

</html>
